primefaces/src/main/resources/META-INF/resources/primefaces/menu/menu.base.js

```diff
@@ -45,6 +44,15 @@ PrimeFaces.widget.Menu = PrimeFaces.widget.BaseWidget.extend({
         }
     },
 
+    /**
+     * Gets the Menu jQuery element.  Override in subclasses to define the menu panel.
+     * @returns {JQuery} The jQuery object for the menu.
+     * @protected
+     */
+    getMenu: function() {
```

Comment:
would really call it getMenuElement

the menu is the whole component itself already

Reply:
Renamed to `getMenuElement()`